railties/lib/rails/generators/rails/app/templates/docker-entrypoint.tt

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash -e
 
+# Enable jemalloc for reduced memory usage and reduce latency.
```

Comment:
Minor typo:

```suggestion
# Enable jemalloc for reduced memory usage and reduced latency.
```

Or just removing is more succint imo:

```suggestion
# Enable jemalloc for reduced memory usage and latency.
```

Reply:
https://github.com/rails/rails/commit/1ca8c8ea3401ec1a88ea5a5adf56b9c65a6746a6